pip_review/__main__.py

```diff
@@ -146,34 +148,25 @@ def latest_version(pkg_name, prerelease=False, silent=False):
     return parse_version(version), version
 
 
-def get_latest_versions(pkg_names, prerelease=False):
-    get_latest = partial(latest_version, prerelease=prerelease, silent=True)
+def get_latest_versions(pkg_names, pre_release=False):
+    get_latest = partial(latest_version, pre_release=pre_release, silent=True)
     versions = map(get_latest, pkg_names)
     return zip(pkg_names, versions)
 
 
-def get_installed_pkgs(local=False):
-    logger = logging.getLogger(u'pip-review')
-    command = 'pip freeze'
-    if packaging_version.parse(pip.__version__) >= packaging_version.parse('8.0.3'):
-        command += ' --all'
+def get_installed_pkgs(local=False, user=False):
+    command = ['pip', 'list', '--format=json']
```

Comment:
Well, using just `pip list` in the latest version shows an error message. You have to set a format option. And the legacy format will be removed in the new version. So, why not already implement that? instead of waiting till the new version and rewrite some code :p